src/dialog-editor/components/modal-field/field.html

```diff
@@ -90,6 +126,12 @@ <h4 class="modal-title" id="myModalLabel" translate>Edit Field Details</h4>
     </dialog-editor-modal-field-template>
     <dialog-editor-modal-field-template ng-switch-when="DialogFieldDropDownList"
                                         template="drop-down-list.html"
+                                        lazy-load="vm.lazyLoad"
+                                        tree-selector-data="vm.treeSelectorData"
+                                        tree-selector-toggle="vm.treeSelectorToggle"
+                                        tree-selector-show="vm.treeSelectorShow"
+                                        tree-selector-include-domain="vm.treeSelectorIncludeDomain"
+                                        on-select="vm.onSelect"
```

Comment:
I really don't like how this is repeated all the times, but it's not part of this PR.